### PR TITLE
chore(smart-contracts): remove custom hardhat error messages

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This OR removes the use of constants to parse hardhat custom error messages (the test helper we use does it already)

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: smart-contracts/test/helpers/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const errors = require('./errors')
```
Comment (julien51) [inline smart-contracts/test/helpers/index.js:1]: Hum, this file does not add much? Why don't we just include the errors file directly?

**Review by julien51 — COMMENTED** [review]
One (silly?) question!

File: smart-contracts/test/helpers/index.js (hunk 1 above)
Comment (clemsos) [reply smart-contracts/test/helpers/index.js:1]: oh it is a common pattern with the other PR that are ongoing (to import functions from a common ./helpers root)

Comment (clemsos) [reply smart-contracts/test/helpers/index.js]: actually was removed by merging

**Review by julien51 — APPROVED** [review]

**Pull request merged**